response/slack/models/pinned_message.py

```diff
@@ -2,39 +2,66 @@
 from django.db import models
 from django.conf import settings
 
-from response.core.models import Incident, ExternalUser, GetOrCreateSlackExternalUser
+from response.core.serializers import ExternalUserSerializer
+from response.core.models import (
+    Incident,
+    ExternalUser,
+    GetOrCreateSlackExternalUser,
+    TimelineEvent,
+)
 
 
 class PinnedMessageManager(models.Manager):
     def add_pin(self, incident, message_ts, author_id, text):
-        name = settings.SLACK_CLIENT.get_user_profile(author_id)['name']
+        name = settings.SLACK_CLIENT.get_user_profile(author_id)["name"]
```

Comment:
I've seen single quotes elsewhere.  Not that it's a blocker, I would like to know what's Python idiomatic :)

Reply:
Hah - unfortunately Python allows both, and both styles are widely used! Even the "official" Python style guide [PEP-8](https://www.python.org/dev/peps/pep-0008/#string-quotes) doesn't make a recommendation.

I use [black](https://github.com/psf/black) as a formatter locally - we should probably standardise on this